rash_book/src/contributing.md

```diff
@@ -57,20 +57,20 @@ This is a rough outline of what a contributor's workflow looks like:
 
 ## Coding Style
 
-Rash projects are written in Rust and follows a functional style trying to keep code simple.
+Rash projects are written in Rust and follow a functional style trying to keep code simple.
 
 ## Comments
 
-Comments should be added to all new methods and structures as is appropriate for the coding
-language. Additionally, if an existing method or structure is modified sufficiently, comments should
-be created if they do not yet exist and updated if they do.
+UYoou should add appropriate comments to all new methods and structures.
```

Comment:
I think this could be `You`? typo error?

Reply:
You're right, it was a typo 😔